index.html

```diff
@@ -13443,10 +13302,6 @@ <h2>Changes since JSON-LD Community Group Final Report</h2>
     <li><a>Term definitions</a> with keys which are of the form of a <a>compact IRI</a> or <a>absolute IRI</a> MUST NOT
       expand to an <a>IRI</a> other than the expansion of the key itself.</li>
     <li>Define different processor modes: <a>pure JSON Processor</a>, <a>event-based JSON processor</a>, and <a>full Processor</a>.</li>
```

Comment:
Should we also remove these processor modes now?

Reply:
We might rename it, but the full Processor mode still defines the ability to extract JSON-LD from html. The PR just removes the need to use this for contexts.

Reply:
k. We at least have a naming collision here then.

[Processing mode](https://w3c.github.io/json-ld-syntax/#dfn-processing-mode) (changed via `@version`)
vs.
[Processor Levels](https://w3c.github.io/json-ld-syntax/#processor-levels) (pure, full, and event-based)